package.json

```diff
@@ -14,39 +14,38 @@
     },
     "dependencies": {
         "@types/debug": "4.1.12",
-        "@types/express": "5.0.2",
-        "@types/node": "14.14.31",
         "@types/socket.io": "3.0.1",
-        "cross-env": "7.0.3",
+        "cross-env": "^10.0.0",
         "debug": "4.4.1",
-        "dotenv": "16.0.0",
+        "dotenv": "^17.2.1",
         "express": "5.1.0",
         "prom-client": "15.1.3",
         "socket.io": "4.8.1",
-        "ts-node-dev": "2.0.0",
-        "typescript": "4.2.3"
+        "ts-node-dev": "2.0.0"
     },
     "license": "MIT",
     "scripts": {
         "build": "tsc",
-        "lint": "npm run lint-fix && eslint '**/*.{js,ts,tsx}'",
-        "lint-fix": "eslint '**/*.{js,ts,tsx}' --fix",
+        "lint": "npm run lint-fix && npx eslint . --ext .js,.ts,.tsx",
```

Comment:
Can you drop lint-fix from here? There is already a dedicated script for it. We should, however, run `tsc --noEmit` as part of linting.